app/src/components/ProfileEdit/QuestionPage.tsx

```diff
@@ -52,122 +56,136 @@ export const QuestionPage = (props: Props) => {
         if (!groups[groupI][subgroupI]) groups[groupI][subgroupI] = []
         groups[groupI][subgroupI].push(state.questions[i])
     }
-
+    
     function setAnswer(index: number, answer: any) {
-        var state2 = {...state}
+        var state2 = { ...state }
         state2.questions[index].answer = answer
+
+        if(answer === "no" && state2.questions[index].showstopper){
```

Comment:
Almost there... Can you make this `if (answer === state2.questions[index].showstopper) {`
Then make the questions with showstoppers have `showstopper: 'no'`
That way, we can have a showstopper for any answer.

Then make sure to resolve the merge conflicts with master.